osd_cluster_create/byo-vpc.adoc

```diff
@@ -0,0 +1,42 @@
+[id="byo-vpc"]
+= Managing a Bring-Your-Own Virtual Private Cloud (VPC)
+:context: byo-vpc
+include::modules/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+Configuring a cluster to include your own pre-configured Virtual Private Cloud (VPC) allows `*[WHAT]*`
+
+.Prerequisites
+* You have a configured VPC
+* You have a configured AWS account in the `osd-staging-2` organization
+
+.Procedure
+
+. Create an IAM user with the name `osdCcsAdmin`.
+.. On this role, attach the `AdministratorAccess` policy.
+.. Generate an access key pair.
+.. Store this generated key pair for use later in this process.
+. Once you have the AWS account ID, run the following command:
++
+[source,terminal]
+----
+osdctl account console -i <$AWS_ACCOUNT_ID> -p osd-staging-2
```

Comment:
What does too `osdctl` used for? How can user get the tool? I didn't find related information in the context.

Reply:
osdctl should not be mentioned here. Its an internal tool, not one customers should use

Reply:
NP, but @fahlmant and @xueli181114  how do I revise these lines:

> Once you have the AWS account ID, run the following command:
> 
> [source,terminal]
> ----
> osdctl account console -i <$AWS_ACCOUNT_ID> -p osd-staging-2
> ----
> 
> [NOTE]
> ====
> You will need AWS credentials for your `osd-staging-2` IAM user in your local `~/.aws/credentials` directory. You can copy these credentials from the same file on the bastion hosts.
> 
> This will generate a temporary console URL for the AWS account.

If I just remove the command, I'm not sure how users get the temporary console URL.

Reply:
They shouldn't need a temporary console url. The customer owns the AWS account, so they can get normal console access.

Reply:
Updated to totally remove this reference.